moncli/entities/column_value/complex.py

```diff
@@ -64,6 +74,34 @@ class TimezoneValue(ColumnValue):
     pass
 
 
-class WeekValue(ColumnValue):
+class WeekValue(ComplexNullValue):
     """A week column value."""
-    pass
\ No newline at end of file
+    
+    native_type = Week
+    allow_casts = (dict)
+    
+    
+    def _convert(self, value):
+        try:
+            start_date = datetime.strptime(value['startDate'], DATE_FORMAT)
+            end_date = datetime.strptime(value['endDate'], DATE_FORMAT)
+            return Week(start=start_date, end=end_date)
+        except (KeyError,ValueError):
+            return {}
+        
+    def _cast(self, value):
+        try:
+            start = value['start']
+            end = value['end']
+            if not (start.tzinfo and end.tzinfo):
+                return Week(start=start,end=end)
+        except (AttributeError,KeyError):
+            raise ColumnValueError('invalid_week_data', self.id, 'Unable to convert "{}" to Week value of end date.'.format(value))
+        
+    def _format(self):
+        try:
+            start_date = self.value.start.date()
+            end_date = self.value.end.date()
+            return {'startDate': str(start_date), 'endDate': str(end_date), 'week_number': self.value.week_number}
```

Comment:
Let's do a datetime.strftime using DATE_FORMAT for these to ensure platform consistency.